README.md

```diff
@@ -7,67 +7,82 @@
+> Change the <distro name> to match the one that you are running. You could see your distro name from the result of the previous command.
+- After your WSL version is updated set it as default by running:
+```PowerShell
+wsl --set-default-version 2
+```
+- Set the chosen distro as default
+```PowerShell
+wsl --set-default <distro name>
+```
+- **Close WSL and reopen it**
+
+After running the above commands. Close PowerShell.
+- Open **Docker Desktop**
+- Navigate to **Settings**
+- From the **General** tab, select **Use WSL 2 based engine**
+
+>You should be all set 🎉
 ---
 
-### Adding your SSH key to the ssh-agent and GitHub
-⚠️ This step is a prerequisite for the installation scripts
+### 1. Add your SSH key to the ssh-agent and GitHub
 
 Here is the guide:
 * [Generate SSH key and add it to agent](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
 
 * [Add key to Github](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)
 
----
-### OSX installation
-
-Install brew and follow the instructions (sometimes it asks you to run some additional commands):
-```shell
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-```
-
-```shell
-curl https://raw.githubusercontent.com/AUBGTheHUB/monolith/master/install_osx.sh | bash
+### 2. Clone the repository to your machine
+```bash
+git clone git@github.com:AUBGTheHUB/monolith.git
 ```
-
-And then run:
+### 3. Navigate to the project directory
 ```bash
-cd ~/go/src/monolith && make post-osx
-
-# make sure to check the output towards the end as certain installations may require manual intervention.
+cd monolith
 ```
----
-### WSL installation
-* For [Ubuntu](https://www.microsoft.com/store/productId/9PDXGNCFSCZV) WSL
+### 4. Open your project in Visual Studio code. Run:
 ```bash
-curl https://raw.githubusercontent.com/AUBGTheHUB/monolith/master/install_wsl.sh | bash
+code .
+```
+### 5. Navigate to Visual Studio Code and perform:
+- <kbd>command</kbd> + <kbd>shift</kbd> + <kbd>P</kbd> (Mac) or <kbd>ctrl</kbd> + <kbd>shift</kbd> + <kbd>P</kbd> (Windows) to open the command palette
+- In the command palette write and select:
+```
+>Dev Containers: Reopen in Container
 ```
 
-And then run:
-```bash
-cd ~/go/src/monolith && make post-wsl
+>After this step you should wait until the container is built. It will install all the dependencies needed for development on its own.
 
-# make sure to check the output towards the end as certain installations may require manual intervention.
-```
+### 6. Check if Dev Container is running successfully
+If your Dev Container is running successfully you should be able to see the following in the bottom-right of your
+Visual Studio Code client.
+
+<img src="https://i.ibb.co/qnYQ6Qp/Screenshot-2024-10-07-at-2-33-55-PM.png" alt="image" border="0">
```

Comment:
What I would suggest that you start doing is upload all images in a `/docs` directory in the repository and point to them with a relative path.

`![image info](docs/image.png)`

Reply:
Fixed in a401561